wrench/src/yaml_frame_writer.rs

```diff
@@ -308,14 +309,10 @@ impl YamlFrameWriter {
         let dl_desc = self.dl_descriptor.take().unwrap();
         let aux_desc = self.aux_descriptor.take().unwrap();
 
-        assert_eq!(data.len(), dl_desc.size() + aux_desc.size() + 4);
+        let payload = Payload::from_data(data);
```

Comment:
is there way for us to still have an assert? It appears that the new code no longer tries to check for the data length